src/docs/introduction/versioning.adoc

```diff
@@ -11,15 +11,16 @@ If necessary a 4th level point release number will be used for successive change
 
 |===
 s| Plugin Version    s| Grails                s| Elasticsearch
+s| 1.4.1           s| 3.1.x                 s| 5.x (tested with 5.4.1)
```

Comment:
Btw, I may not agree with this versioning policy but didn't want to change it since I'm new to the project and also I don't know how recent was the last policy change.